real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/RBSMNode.java

```diff
@@ -299,6 +301,21 @@ public void actionPerformed(String topicName, Message m) {
 					commandedBrakeEngaged = ((BrakeControlMessage)m).isBrakeEngaged();
 				}
 			});
+			new Subscriber(NodeChannel.ENCODER_RESET.getMsgPath(), new MessageListener() {
+				@Override
+				public void actionPerformed(String topicName, Message m) {
+					byte[] bytes = new byte[6];
+					bytes[0] = (byte)RBSerialMessage.getHeaderByte("RBSM_MID_ENC_RESET_REQUEST"); //Reset request header
+					bytes[1] = (byte)0;
+					bytes[2] = (byte)0;
+					bytes[3] = (byte)0;
+					bytes[4] = (byte)(0xc0);  
```

Comment:
why is this C0? I would prefer for us to not have magic values in the communication system